src/components/InterestForm/styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const FormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 60%;
+  float: right;
+  margin-top: 1rem;
+  gap: 0.5rem;
+`;
+
+export const FormTextArea = styled.textarea`
```

Comment:
I would suggest adding a `resize: vertical` property to allow the user to resize the `textarea` component. Though this is a change that is dependent on the design. I think it is preferable to improve user experience, especially given that we are already accounting for a vertically overflowing card, but we may want to consult @davidqing6432 ?

Reply:
yeah, I think this change should be up to David since it messes with the design a bit.

Reply:
yeah we can run by david tmr